content-src/experiments/price-scout.yaml

```diff
@@ -0,0 +1,53 @@
+addon_id: shopping-testpilot@mozilla.org
+bug_report_url: "https://github.com/mozilla/price-scout/issues"
+contributors:
+  -
+    avatar: /static/images/experiments/price-scout/avatars/leeroy-avatar.png
+    display_name: "Leeroy Jenkins"
+created: "2018-08-07T13:00:00.00Z"
+description: "Aliquam quis orci id ex malesuada congue. Nullam fringilla nulla rutrum, placerat elit et, sodales urna."
+details:
+  -
+    copy: "Maecenas rutrum sapien diam, vestibulum posuere odio ornare pellentesque."
+    image: /static/images/experiments/price-scout/details/detail1.jpg
+  -
+    copy: "Donec aliquet aliquet augue sed viverra."
+    image: /static/images/experiments/price-scout/details/detail2.jpg
+  -
+    copy: "Sed viverra ligula ex, ut pellentesque mauris maximus id."
+    image: /static/images/experiments/price-scout/details/detail3.jpg
```

Comment:
May possibly want to add `dev: true` for some added footgun protection until we're closer to launch.